metric.go

```diff
@@ -26,7 +26,7 @@ import (
 // instrumentationName is empty, then a implementation defined default name
 // will be used instead.
 //
-// This is short for MeterProvider().Meter(name)
+// This is short for otel.GetMeterProvider().Meter(name)
```

Comment:
These changes have been resolved upstream. We can revert changes to this file when synced.